backend/tilavarauspalvelu/api/graphql/types/reservation/serializers/approve_serializers.py

```
@@ -58,7 +59,7 @@ def validate(self, data: ReservationApproveData) -> ReservationApproveData:
     def update(self, instance: Reservation, validated_data: ReservationApproveData) -> Reservation:
         if self.instance.access_type == AccessType.ACCESS_CODE and instance.recurring_reservation is None:
             # Allow activation in Pindora to fail, will be handled by a background task.
-            with suppress(Exception):
+            with suppress(PindoraClientError):
```

All exceptions are suppressed so that all errors in the `requests` library are also suppressed. This could happen when Pindora is down, or we otherwise couldn't get a connection to them. Approval (and other operations) should still work in these situations.

---

Maybe should coerce all exceptions from requests to some `ExternalClientError` and suppress those instead.

---

Fixed 🛠️